src/model/bpmn/shape/ShapeBpmnElementKind.ts

```diff
@@ -24,8 +24,21 @@ export enum ShapeBpmnElementKind {
   TASK = 'task',
   GATEWAY_PARALLEL = 'parallelGateway',
   GATEWAY_EXCLUSIVE = 'exclusiveGateway',
+  GATEWAY_INCLUSIVE = 'inclusiveGateway',
```

Comment:
:warning: to be discussed, with the current implementation, these added types should be removed because the related tests are commented. In addition
- this is not consistent as all BPMN elements supporting default sequence flows (for instance, `manual` or `business rules` tasks)
- this implies changing the documentation that explains the work to do to support new BPMN elements

Reply:
Let's talk about it tomorrow :)

Reply:
As we discussed, I'll add all missing kind added in the default sequence flow tests, and comment them + add comment for uncomment.

Reply:
Done